bigquery/google/cloud/bigquery/table.py

```diff
@@ -783,22 +783,38 @@ def __init__(self, values, field_to_index):
         self._xxx_field_to_index = field_to_index
 
     def values(self):
+        """Return the values included in this row.
+
+        Returns:
+            Sequence[object]: A sequence of length ``len(row)``.
+        """
         return self._xxx_values
 
     def keys(self):
-        """
-        Return keys as of a dict:
-        >>> Row(('a', 'b'), {'x': 0, 'y': 1}).keys()
-        ['x', 'y']
+        """Return the keys for using a row as a dict.
+
+        Returns:
+            Sequence[str]: The keys corresponding to the columns of a row
+
+        Examples:
+
+            >>> Row(('a', 'b'), {'x': 0, 'y': 1}).keys()
+            ['x', 'y']
         """
         keys = self._xxx_field_to_index.keys()
         return keys
```

Comment:
Where you're refactoring, just drop the `keys =` and return the output? Also, do you want `six.iterkeys`?